rust/nasl-builtin-cryptographic/tests/hash.rs

```diff
@@ -0,0 +1,169 @@
+// SPDX-FileCopyrightText: 2024 Greenbone AG
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+mod helper;
+
+#[cfg(test)]
+mod tests {
+    use nasl_interpreter::*;
+
+    #[test]
+    fn hash_md5() {
+        let code = r#"
+        a = MD5("hola mundo");
+        hexstr(a);
+        "#;
+        let mut register = Register::default();
+        let binding = ContextBuilder::default();
+        let context = binding.build();
+        let mut interpreter = Interpreter::new(&mut register, &context);
+        let mut parser =
+            parse(code).map(|x| interpreter.resolve(&x.expect("no parse error expected")));
+        assert_eq!(
+            parser.next(),
+            Some(Ok(NaslValue::Data(
+                [10, 208, 102, 165, 210, 159, 63, 42, 42, 28, 124, 23, 221, 8, 42, 121].to_vec()
+            )))
+        );
+        assert_eq!(
+            parser.next(),
+            Some(Ok(NaslValue::String(
+                "0ad066a5d29f3f2a2a1c7c17dd082a79".to_string()
+            )))
+        );
+    }
```

Comment:
Why testing it twice?

Reply:
Becasue I fixed/changed the hexstr for data type. Moved the test to the right place

Reply:
fixed with [6228f14](https://github.com/greenbone/openvas-scanner/pull/1616/commits/6228f14b648b7a78aaa8a52392ade84bcf5f38e6)